myia/parser.py

```diff
@@ -77,14 +78,17 @@ def __init__(self, msg, loc):
 _parse_cache = {}
 
 
-def parse(func):
+def parse(func, parser_opt=False):
```

Comment:
I'm of the opinion that parser_opt should be always on.

I fail to see a case where one would want to genuinely turn it off.